accessibility/README.md

```diff
@@ -0,0 +1,495 @@
+* `aria-label` is another approach for labelling elements, and can be used when there is no visible element in the DOM that can be used directly as a label or referenced with `aria-labelledby`
+* **Use `aria-label` with caution** - if adding `visually-hidden` text is possible, this is preferred instead
+  * `aria-label` has a rocky history with screen reader support with certain combinations of screen readers and browsers, whereas printed text is always read out (unless `display: none` or `visibility: hidden` styles or the `hidden` or `aria-hidden` attributes are used)
+  * Using `visually-hidden` text that is rendered in the DOM can also benefit users users that are translating the language of your site as these services are often better at detecting `visually-hidden` text then they are `aria-label` attributes
+  * See [this blog post from 2020](https://gomakethings.com/revisting-aria-label-versus-a-visually-hidden-class/)
+* Examples of using this are similar to adding a `<span class="visually-hidden">` element
+
+```vue
+<button
+  type="button"
+  aria-label="Toggle Menu"
+  @click="openMenuDrawer"
+>
+  <icon-menu />
+</button>
+
+<a href="/account" aria-label="Account">
+  <icon-account />
+</a>
+```
+
+[ꜛ Back to TOC](#table-of-contents)
+
+## Alt Text
+* Alt text is extremely useful for ensuring that images on the site can be used by people with various disabilities.
+* Images can be categorised in different ways and there are some different recommendations for alt text depending on the type of image.
+* The W3C Web Accessibility Initiative has some good resources around Images:
+  * [Images Tutorial](https://www.w3.org/WAI/tutorials/images/) - Outlines the different purposes an image might have and recommendations for adding accessible text
+  * [`alt` Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) - Provides a helpful decision tree on how to handle different situations with images, including when an empty `alt` attribute is appropriate. 
+
+[ꜛ Back to TOC](#table-of-contents)
+
+## Element IDs
+
+* A variety of accessibility code and functionality (such as the `label` element's `for` attribute, `aria-labelledby` and other `aria-` attributes) use the `id` attribute to reference other elements
+  * For example, linking a checkbox input with a label using `id`/`for` allows the user to click on the label to toggle the checkbox
+  * This helps make the input easier to trigger with a mouse, and also makes it possible to style custom inputs (for example custom checkboxes) by applying styles to the `<label>` element
+* When writing components that use the `id` attribute - ensure that the `id` will always be unique
+  * Avoid hard coding an id for an element unless you know it will only be used once
+  * This is particularly important when the component is designed to be re-used, for example a swatch or a line item component
+  * Unique ids can be generated using passed in data, and additionally namespaces if needed
```

Comment:
```suggestion
  * Unique IDs can be generated using passed in data, and additionally namespaces if needed
```
This line is more referring to the concept of 'IDs' rather than the attribute `id`, hence the different casing.